configure.ac

```diff
@@ -308,7 +308,7 @@ AC_MSG_RESULT([$LIBS])
 # Dependencies that themselves have a pkg-config file available.
 #
 PC_REQUIRES=""
-AS_IF([test "$pmix_hwloc_support_will_build" = "yes"],
+AS_IF([test "$pmix_hwloc_source" != "cobuild"],
```

Comment:
Probably want to keep the "will_build" check along with the "cobuild" one - we shouldn't mandate an hwloc dependency if we are not building support for it in PMIx